src/containerapp/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 upcoming
 ++++++
+* 'az containerapp add-on chroma': support for chroma add-on
```

Comment:
As version `0.3.44` has been released, please move the description to the `upcoming` part.

Reply:
Will do will do. thank you.